content/2019/Making-things-flow/index.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "Making Things Flow"
+publishDate: 2019-12-01
+draft: true
+description: "Making the behaviour of Vim flow"
+author:
+  github: "george-b"
+  name: "George Brown"
+---
+
+# Making things flow
+
+Changing the behaviour of one's editor is a common subject, but what if the desired behaviour can't be achieved by configuring a setting offered by your editor? Thankfully Vim is an editor which allows powerful scripting by its users.
+
+## Look Ma, no mappings!
+
+There's nothing wrong with mappings. The purpose of this article is to explore more generalised methods whereas mappings generally have a more singular focus. Or to put is another way this article will focus more on the behaviour and response of the editor rather than defining singular imperative actions.
```

Comment:
```suggestion
There's nothing wrong with mappings. The purpose of this article is to explore more generalised methods whereas mappings generally have a more singular focus. Or to put it another way this article will focus more on the behaviour and response of the editor rather than defining singular imperative actions.
```